src/sirtextstyle.c

```diff
@@ -215,3 +217,43 @@ bool _sir_setcolormode(sir_colormode mode) {
 
     return true;
 }
+#else /* SIR_NO_TEXT_STYLING */
+const char* _sir_gettextstyle(sir_level level) {
+    SIR_UNUSED(level);
```

Comment:
I think it would be better to also `SIR_ASSERT(false)` before returning NULL, in case some naive caller doesn't expect a NULL retval. Or alternatively, return `""`.

Reply:
Let's SIR_ASSERT(false);